schema/canonical/crc_test.go

```diff
@@ -1,24 +1,23 @@
 package canonical
 
 import (
-	"testing"
```

Comment:
This is a deliberate convention - stdlib imports are in a separate block from 3rd party imports. `testing` and `encoding/hex` are stdlib, but testify is a 3rd party library.

Reply:
I use goland as IDE and it have tendency to do things behind my back

Reply:
I will revert